static/scss/partials/modals.scss

```diff
@@ -0,0 +1,151 @@
+.modal-message {
+    @include text-body-sm;
+    color: $color-grey-60;
+    margin: 0 auto $spacing-sm auto;
+
+    strong {
+        word-break: break-word;
+    }
+    
+    &.warning-text {
+        color: $color-error;
+    }
+}
+
+.alias-to-delete {
+    display: inline;
+    word-break: break-all;
+
+    &.demi {
+        display: inline-block;
+        margin: 0 auto $spacing-md;
+        // margin: 0 auto $spacing-md 0;
+        color: $color-grey-60;
+        max-width: $content-xs;
+        text-overflow: ellipsis;
+        overflow: hidden;
+        word-wrap: unset;
+    }
+}
+
+// Modal Protocol Refactor
+.c-modal-wrapper {
+    position: fixed;
+    display: none;
+    background: rgba(0, 0, 0, .4);
```

Comment:
Should this be a variable?

Reply:
Don't think so, alternatively to incorporate Protocol I think you could use the `::before` selector and then set that to a `$color-black` background and `opacity: 0.4`

Reply:
I can also use the SCSS `rgba($base-color, 0.88)` [function](https://sass-lang.com/documentation/variables). I'll do that.